.buildkite/scripts/run_standard_models.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+## remove failure for testing
+set -euo pipefail
+
+apt-get update
+apt-get install libsasl2-dev
+
+python3 -m venv venv
+. venv/bin/activate
+pip install --upgrade pip setuptools
+pip install -r integration_tests/requirements.txt
+mkdir -p ~/.dbt
+cp integration_tests/ci/sample.profiles.yml ~/.dbt/profiles.yml
+
+db=$1
+echo `pwd`
+cd integration_tests
+dbt deps ## Install all packages needed
+
+shift ## Skips the first argument (warehouse) and moves to only looking at the data model arguments
+
+for model in "$@" ## Iterates over all non warehouse arguments
+do
+    echo -e "\ncompiling "$model"\n"
+    cd dbt_packages/$model/integration_tests/
+    dbt deps
+    cp ../../../packages_ft_utils_override.yml packages.yml
+    dbt deps
+    if [ "$model" = "linkedin" ]; then
+        value_to_replace=$(grep ""$model"_ads_schema:" dbt_project.yml | awk '{ print $2 }')
```

Comment:
"$model" is needed because that is how we fetch the component of the for loop. Here we are just checking if the component of the loop is equal to linkedin.